### ISSUE TITLE
Found an unknown BlockId:Meta 44:48

### ISSUE THREAD

**Issue by joserobjr** [PowerNukkit/PowerNukkit#410]
# 🐞 I found a bug

I started a very fresh world, joined in and got this error in the log.

### ▶ Steps to Reproduce

1. Load the seed 1595680813690
2. Join the world
3. See the logs

### ✔ Expected Behavior

Do not get replaced with an UPDATE! block (I couldn't find it in the world yet)

### ❌ Actual Behavior

Becomes an UPDATE! block

### 📋 Debug information

* PowerNukkit version: 1.3.1.0-PN
* Debug link: https://hastebin.com/icavayuwud
* Timings link (if relevant): 

### 💢 Crash Dump, Stack Trace and Other Files

```java
PS C:\Users\joser\tutorial-docker\PowerNukkit> docker run -it --name power -p 19132:19132/udp powernukkit:1.3.0.1-PN
12:40:10 [INFO ] Welcome! Please choose a language first!
12:40:10 [INFO ] eng => English
12:40:10 [INFO ] chs => 中文（简体）
12:40:10 [INFO ] cht => 中文（繁體）
12:40:10 [INFO ] jpn => 日本語
12:40:10 [INFO ] rus => Pyccĸий
12:40:10 [INFO ] spa => Español
12:40:10 [INFO ] pol => Polish
12:40:10 [INFO ] bra => Português-Brasil
12:40:10 [INFO ] kor => 한국어
12:40:10 [INFO ] ukr => Українська
12:40:10 [INFO ] deu => Deutsch
12:40:10 [INFO ] ltu => Lietuviškai
12:40:10 [INFO ] idn => Indonesia
12:40:10 [INFO ] cze => Czech
12:40:10 [INFO ] tur => Turkish
12:40:10 [INFO ] fin => Suomi
> bra
12:40:12 [INFO ] Loading nukkit.yml ...
12:40:12 [INFO ] Loading server.properties ...
12:40:12 [INFO ] Selecionado Português (bra) como a linguagem base
12:40:12 [INFO ] Iniciando Servidor de Minecraft: BE, versão v1.16.0
12:40:12 [INFO ] Selected Zlib Provider: 2 (cn.nukkit.utils.ZlibThreadLocal)
12:40:12 [INFO ] Abrindo servidor em 0.0.0.0:19132
12:40:12 [INFO ] Este servidor está rodando Nukkit versão 1.3.1.0-PN (git-079d141) "PowerNukkit" (API 1.0.11)
12:40:12 [INFO ] Nukkit é distribuido com a licença LGPL
12:40:13 [INFO ] Loading recipes...
12:40:13 [INFO ] Loaded 1509 recipes.
12:40:13 [INFO ] Successfully loaded 0 resource packs
12:40:13 [WARN ] Mundo "world" não encontrado
12:40:13 [INFO ] Preparando mundo "world"
12:40:13 [WARN ] Mundo "nether" não encontrado
12:40:13 [INFO ] No level called "nether" found, creating default nether level.
12:40:13 [INFO ] Preparando mundo "nether"
12:40:13 [INFO ] Ativando PowerNukkit v1.3.1.0-PN
12:40:13 [INFO ] Iniciando listener de status GS4
12:40:13 [INFO ] Porta de query está ativada em 19132
12:40:13 [INFO ] Query rodando em 0.0.0.0:19132
12:40:13 [INFO ] Modo de jogo padrão: Modo Sobrevivência
12:40:13 [INFO ] Finalizado (3.416s)! Para ver a ajuda, escreva "help" ou "?"
12:41:34 [INFO ] Data não encontrada do player "24693216-4211-36d2-a5c2-0479c49afe98", criando novo perfil
12:41:35 [INFO ] Jose Roberto YT[/172.17.0.1:48895] entrou com o id de entidade 1 em (world, 129.7919, 70.0, 127.6653)
12:41:37 [ERROR] Found an unknown BlockId:Meta combination: 44:48, replacing with an "UPDATE!" block.
12:41:45 [INFO ] Jose Roberto YT entrou no jogo
> op "Jose Roberto YT"
12:42:37 [INFO ] [CONSOLE: Jose Roberto YT virou operador(a)]
> gamemode 1 "Jose Roberto YT"
12:42:48 [INFO ] [CONSOLE: Mudado o modo de jogo de Jose Roberto YT para Modo Criativo]
> seed
12:43:25 [INFO ] Seed: 1595680813690
>
```

**Comment by pinkpig58** [comment]
It might have something to do with this: https://github.com/CloudburstMC/Nukkit/commit/eb0a4e6006a534ee8dea6c077e7ca7add5ead455

**Comment by joserobjr** [comment]
This is the world that generated the log message:
[PowerNukkit-Issue-410.zip](https://github.com/PowerNukkit/PowerNukkit/files/4977214/PowerNukkit-Issue-410.zip)

@pinkpig58 Thank you, but I don't think that commit could cause this issue, PowerNukkit uses a different `GlobalBlockPalette` class to support high block ids and data.
https://github.com/PowerNukkit/PowerNukkit/blob/bbe42c48e38d1609fcb3fc4880f16cbeaef3e1a7/src/main/java/cn/nukkit/level/GlobalBlockPalette.java#L126-L145

 It's something funky happening during world generation.

**Comment by joserobjr** [comment]
Maybe it was this mesa biome bug:
https://github.com/PowerNukkit/PowerNukkit/pull/287/commits/72d2eb29b7d67fe4685cbf636758eda8f933ea7a#diff-737ee3c6c1dff66f29e1f9c543741725R99

**Comment by joserobjr** [comment]
Indeed, it has a mesa biome near the spawn:
![image](https://user-images.githubusercontent.com/3679022/88473296-6fb8a200-cef2-11ea-888d-930645fc8635.png)

**Comment by joserobjr** [comment]
Found the update blocks:
![image](https://user-images.githubusercontent.com/3679022/88473364-29b00e00-cef3-11ea-8261-d8e50f5d96ed.png)

**Comment by joserobjr** [comment]
No longer happening in 1.3.1.2-PN-SNAPSHOT. I think it was fixed by #287 at https://github.com/PowerNukkit/PowerNukkit/pull/287/files#diff-737ee3c6c1dff66f29e1f9c543741725L99-R99

![image](https://user-images.githubusercontent.com/3679022/89721133-472fae00-d9b0-11ea-96a1-da1b602f1205.png)
